Library/ViewModels/PledgeViewModel.swift

```diff
@@ -65,11 +80,15 @@ public protocol PledgeViewModelOutputs {
   var descriptionViewHidden: Signal<Bool, Never> { get }
   var goToApplePayPaymentAuthorization: Signal<PaymentAuthorizationData, Never> { get }
   var goToThanks: Signal<Project, Never> { get }
+  var notifyDelegateUpdatePledgeDidSucceedWithMessage: Signal<String, Never> { get }
   var paymentMethodsViewHidden: Signal<Bool, Never> { get }
+  var popViewController: Signal<(), Never> { get }
```

Comment:
So something I noticed while testing is it seems like the `popViewController` and `notifyDelegateUpdatePledgeDidSucceedWithMessage` outputs are firing at the same time, which means that the pop animation and the message banner animation overlap. I think it would be _nice_ to only show the message banner once the pop animation has completed. 🤔 Maybe setting a delay on the `notifyDelegateUpdatePledgeDidSucceedWithMessage` signal could help? I'm not sure if it's worth the trouble but just a thought.

Reply:
I've added this to the JIRA ticket in which we'll address the refreshing of that view.